clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -34,6 +36,8 @@ import org.ossreviewtoolkit.clients.fossid.model.rules.RuleType
 internal const val SCAN_GROUP = "scans"
 private const val PROJECT_GROUP = "projects"
 
+private val logger = loggerOf(FossIdRestService::class.java)
```

Comment:
Can't you use `FossIdRestService.logger` directly? Or first move it from the companion object to a top-level property for more convenient use over here?